HabitTracker/Menu.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace HabitTracker
+{
+    public class Menu
+    {
+        public static void ShowMenu()
```

Comment:
🟢 **Menu UI**

💡 Very novel menu implementation I have not seen a student use before. Great job! 🥇